python/restate/context.py

```diff
@@ -79,6 +79,19 @@ def clear(self, name: str) -> None:
     def clear_all(self) -> None:
         """clear all the values in the store."""
 
+@dataclass
+class RunExponentialBackoffRetryConfig:
+    """
+    Exponential backoff Retry Configuration.
+
+    Attributes:
+        initial_interval: The initial interval in milliseconds to wait before the first retry.
+        max_attempts: The maximum number of attempts to retry.
+        max_duration: The maximum duration in milliseconds to retry.
+    """
```

Comment:
FYI this is the docs text I used in the other SDKs: https://github.com/restatedev/sdk-java/pull/375/files#diff-c7114c260c8e7e1653409b43f92d24cfd46e51517bf2ed166871dc963de54ff0R16